lib/recipe-repo.js

```diff
@@ -22,15 +22,8 @@ const sequenceSchema = joi.object().keys({
     .unique((a, b) => Object.keys(a)[0] === Object.keys(b)[0])
     .required()
     .items(joi.object().length(1)),
-  executionDelay: joi.alternatives().conditional("namespace", {
```

Comment:
Oh, we used this in lu-broker mostly for when a sub-sequence ran "too quickly" for tests to work. Don't think it would help the current flakiness though, so good to get rid of it from the schema 👍

Reply:
Well, it was never used in either the pubsub or cloud tasks version of b0rker, so it should be safe to remove. We can always add it back.